pages/getting-started/writing-your-first-test.md

```diff
@@ -125,3 +125,149 @@ Expect vem do [Chai](http://www.chaijs.com/).
 alguma familiaridade e conhecimento prévio. Mas se não tiver, tudo bem.
 
 ## Escreva um teste real
+
+Um teste sólido geralmente cobre 3 fases:
+
+1. Configurar o estado da aplicação
+2. Realizar uma ação
+3. Fazer uma asserção sobre o resultado da ação
+
+Também conhecido pela tríade "Given, When, Then", ou "Arrange, Act, Assert". Mas a ideia é: 
+primeiro a aplicação é colocada em um estado específico, então uma ação é realizada na 
+aplicação, o que causa uma mudança, e finalmente o resultado da apicação é checado.
+
+Hoje, vamos ver esses passos de perto e mapeá-los claramente em comandos do Cypress:
+
+1. Visitar uma página web.
+2. Consultar um elemento.
+3. Interagir com o elemento.
+4. Verificar o conteúdo da página.
+
+### Passo 1: Visitar uma página web
+
+Primeiro, vamos visitar uma página web. Neste exemplo vamos visitar a aplicação 
+[Kitchen Sink](https://docs.cypress.io/examples/examples/applications.html#Kitchen-Sink)
+para que você possa experimentar o Cypress sem se preocupar em procurar uma página para 
+testar.
+
+Podemos passar a URL que queremos visitar para o `cy.visit()`. Vamos substituir nosso teste 
+anterior pelo teste abaixo que de fato visita uma página:
+
+```javascript
+describe('My First Test', () => {
+  it('Visits the Kitchen Sink', () => {
+    cy.visit('https://example.cypress.io')
+  })
+})
+```
+
+Salve o arquivo e retorne ao Cypress Test Runner. Você vai perceber algumas coisas:
+
+1. O [Log de Comandos](https://docs.cypress.io/guides/core-concepts/test-runner.html#Command-Log)
+agora exibe a nova ação `visit`.
+2. A aplicação Kitchen Sink foi carregada no painel de 
+[pré visualização da aplicação](https://docs.cypress.io/guides/core-concepts/test-runner.html#Overview).
+3. O teste está verde, mesmo não tendo nenhuma asserção.
+4. O `visit` exibe um estado pendente em azul até que a página termine de carregar.
+
+Se essa solicitação tivesse retornado com um código de status diferente de 2xx, como 404 ou 500, 
+ou se houvesse um erro de JavaScript no código do aplicativo, o teste teria falhado.
+
+[Veja o exemplo acima neste vídeo](https://docs.cypress.io/img/snippets/first-test-visit-30fps.mp4).
```

Comment:
what do you think to make consistent and on every vídeo we can type "Vídeo de exemplo." like on the first video in this section. Makes sense?

Reply:
Fixed